src/tracks/ui/EditCursorOverlay.cpp

```diff
@@ -29,14 +29,15 @@ namespace {
       return (m >= l && m < h);
    }
 }
-
+#ifndef EXPERIMENTAL_NOTETRACK_OVERLAY
```

Comment:
I do not understand the changes in this file.  This disables the vertical stroke to indicate the zero-length selection.  Why?  And why do you use that particular EXPERIMENTAL?  Where is this change specified?

Can this be another small commit?

Reply:
This is an error, thanks. Still i did some changes, which allow me to draw cursor over affordance area properly

Reply:
I see the change now at 6888d9006ff9286ec38e8dd1c3e3c0324240269a, but I don't yet approve that.

Reply:
Is there a reason you can't implement the change in this file, rather than in TrackPanel?